library/lua/3rdparty/alt_getopt.lua

```diff
@@ -0,0 +1,169 @@
+-- Copyright (c) 2009 Aleksey Cheusov <vle@gmx.net>
+--
+-- Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
If you could also add an entry for this in `docs/LICENSE.rst`, that would be great.

Reply:
done. sorry for forgetting that, especially after I explicitly added the others for xlsxio...